.github/workflows/release.yml

```
@@ -8,7 +8,7 @@ env:
   DOTNET_NOLOGO: true
 jobs:
   release:
-    runs-on: ubuntu-20.04
+    runs-on: windows-2019
```

Need some comment so we remember later why we're deviating from the "standard"
```suggestion
    runs-on: windows-2019 # Windows required for ILMerge to work in ScriptBuilderTask
```